### ISSUE TITLE
Please add filters for date, type, length, and quick properties.

### ISSUE THREAD

**Issue by sapioit** [sabrogden/Ditto#677]
Being able to filter things based on the date, type, length, and quick properties, of the clips, would make it much easier to find clips, especially for those who have a lot of clips.

**Comment by schmurtzm** [comment]
Yes this is really a shame that we can't display the date and use it to find an old clip (it is present in the database so it could be nice to have the possibility to use it). 

The filters are a great idea but the the UI should also have an option to display the date too: in other words having columns in the UI could be awesome (date, type, lenght...) .

Edit: the "Delete Clip Data" window already contains contains most of these elements. 
Note: the ["Everything" syntax](https://www.voidtools.com/support/everything/searching/) could be used for Ditto to be able to search by date and other parameters.

**Comment by sapioit** [comment]
The syntax of Everything Search is nice, but it might be more difficult to add, than GUI elements for filters, which to show up when pressing a single button for Advanced Features.

**Comment by Chealer** [comment]
@schmurtzm:
> The filters are a great idea but the the UI should also have an option to display the date too: in other words having columns in the UI could be awesome (date, type, lenght...) .

A column for the datetime was suggested in ticket #984.

> Edit: the "Delete Clip Data" window already contains contains most of these elements.

Indeed, as shown in [that comment](https://github.com/sabrogden/Ditto/issues/875#issuecomment-2983877391). Thanks
